plugins/imjournal/imjournal.c

```diff
@@ -698,13 +715,20 @@ CODESTARTrunInput
 			continue;
 		}
 
+		/*
+		 * update journal disk usage before reading the new message.
+		 */
+		if (sd_journal_get_usage(j, (uint64_t *)&statsCounter.diskUsageBytes) < 0) {
+			LogError(0, RS_RET_ERR, "imjournal: sd_get_usage() failed");
```

Comment:
we shoud add the -result of sd_journal_get_usage to errmsg so that we get a better error messag

```
const int e = sd_jour...
....
LogError(e, RS_...)
```